CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -530,7 +540,7 @@ Pwn DC:				120
 
     		The KRO-W5 or "Crow" is a lightweight quad-rotor drone 
 designed for low-altitude operation. Combining agility with modularity. Its
-onboard targeting laser gives the ally nearest to it a +10 bonus to accuracy 
+onboard targeting laser gives the ally nearest to it a +2 bonus to accuracy 
```

Comment:
Extension to the above query about effective doubling of accuracy bonus.

Reply:
Intentional buff